test/integration/scheduler/priorities_test.go

```diff
@@ -441,7 +441,7 @@ func TestPodTopologySpreadScoring(t *testing.T) {
 		{
 			name: "place pod on a ~0~/1/2/3 cluster with MaxSkew=1, node-1 is the preferred fit",
 			incomingPod: st.MakePod().Namespace(ns).Name("p").Label("foo", "").Container(pause).
-				SpreadConstraint(1, "node", softSpread, st.MakeLabelSelector().Exists("foo").Obj(), nil).
+				SpreadConstraint(1, "node", softSpread, st.MakeLabelSelector().Exists("foo").Obj(), nil, nil, nil).
```

Comment:
Add a couple of scoring tests for the new feature.

Reply:
Sorry, add scoring tests at wrong places. Done :)